s4e-backend/src/main/java/pl/cyfronet/s4e/data/repository/query/QueryBuilderImpl.java

```diff
@@ -15,7 +15,7 @@ public QueryBuilderImpl() {
         columns.add(Scene.COLUMN_ID);
         columns.add(Scene.COLUMN_PRODUCT_ID);
         columns.add(Scene.COLUMN_SCENE_KEY);
-        columns.add("ST_AsText(ST_Transform(" + Scene.COLUMN_FOOTPRINT + ",4326)) AS footprint");
+        columns.add("ST_AsText(ST_Transform(" + Scene.COLUMN_FOOTPRINT + ",4326),5) AS footprint");
```

Comment:
What mean value `, 5` in this context?

Reply:
It's the maximum numer of decimal digits, the same as I use in java format. I add this parameter here to keep the output generated by postgis (in search endpoints) and java (in scene endpoint) consistent.
See https://postgis.net/docs/ST_AsText.html for the postgis reference.